app/assets/javascripts/short-form/ShortFormApplicationService.js.coffee.erb

```diff
@@ -321,6 +322,187 @@ ShortFormApplicationService = ($localStorage, $translate, $http, Upload, uuid) -
       Service.preferences["#{fileType}_error"] = true
     ))
 
+  Service.submitApplication = (listingId)->
+    applicationParams = Service._formatApplication(listingId)
+    console.log(applicationParams)
+    $http.post('/api/v1/short-form/submit-application', applicationParams).success((data, status, headers, config) ->
+      data
+    ).error( (data, status, headers, config) ->
+      return
+    )
+
+  Service._formatApplication = (listingId) ->
```

Comment:
I'll probably want to pull this out into it's own helper/service in the future, especially when we receive app info -- but kept it here for now so as not to setup a new test file.